### PR TITLE
fixing Henon help

### PR DESCRIPTION
**Pull request by tiagmoraismorgado**

### CODE REVIEW

**Comment by telephon** [comment]
Thanks again for this. I feel that for the core helpfiles, a minimalist style (without `~mouseSomething`) is really better.

**Comment by crucialfelix** [comment]
Yes, I think the simplistic style of {}.play is better. The mind should only be thinking about the example, not how the example is written. Each example should be the same style.

File: HelpSource/Classes/HenonC.schelp [hunk 1]
```diff
@@ -43,17 +43,13 @@ Second value of x
 examples::
 code::
 // default initial params
-~defaultParms={ HenonC.ar(MouseX.kr(20, SampleRate.ir)) * 0.2 !2};
+{ HenonC.ar(MouseX.kr(20, SampleRate.ir)) * 0.2 !2}.scope(s);
```
Comment (telephon) [inline HelpSource/Classes/HenonC.schelp]: better formatting: `0.2 ! 2 }.scope(s);` (spaces between operands and operators)

File: HelpSource/Classes/HenonC.schelp [hunk 2]
```diff
@@ -43,17 +43,13 @@ Second value of x
...
 
 // mouse-control of params
-~mousControl={ HenonC.ar(SampleRate.ir/4, MouseX.kr(1,1.4), MouseY.kr(0,0.3)) * 0.2 !2};
+{ HenonC.ar(SampleRate.ir/4, MouseX.kr(1,1.4), MouseY.kr(0,0.3)) * 0.2 !2}.scope(s);
 
 // randomly modulate params
-~randModPars=({ HenonC.ar(SampleRate.ir/8, LFNoise2.kr(1, 0.2, 1.2), LFNoise2.kr(1, 0.15, 0.15)) ! 2 * 0.02 });
+({ HenonC.ar(SampleRate.ir/8, LFNoise2.kr(1, 0.2, 1.2), LFNoise2.kr(1, 0.15, 0.15)) ! 2 * 0.02 }).scope(s);
```
Comment (telephon) [inline HelpSource/Classes/HenonC.schelp]: no need for the round parentheses, you don't have them in the above.
also better to leave out the `s`, it is redundant.

`{ HenonC.ar(SampleRate.ir/8, LFNoise2.kr(1, 0.2, 1.2), LFNoise2.kr(1, 0.15, 0.15)) ! 2 * 0.02 }.scope`

**Pull request merged**